com.unity.shadergraph/Editor/Drawing/Blackboard/SGBlackboard.cs

```diff
@@ -47,6 +48,21 @@ public void OnControllerEvent(SGControllerEvent e)
         {
         }
 
+        public void SetCurrentVariantUsage(int currentVariantCount, int maxVariantCount)
+        {
+            if(currentVariantCount < maxVariantCount &&  m_VariantExceededHelpBox != null)
```

Comment:
need a space between the "if" and the "(".  Guessing this is tripping the formatter.

Reply:
Didn't even remember to check formatting on that first pass :)